client/app/bundles/course/leaderboard/pages/LeaderboardIndex/index.tsx

```diff
@@ -159,7 +161,12 @@ const LeaderboardIndex: FC<Props> = (props) => {
             columnSpacing={2}
             container
             direction="row"
-            display={tabValue === 'leaderboard-tab' ? 'flex' : 'none'}
+            display={
+              tabValue === 'leaderboard-tab' ||
+              settings.groupleaderboardTitle === undefined
```

Comment:
Is there any reason why we need to add the additional check `settings.groupleaderboardTitle === undefined` here?

Reply:
it's to handle the following cases:
1. when activating the group leaderboard, we choose the Group Leaderboard Tab (hence the tabValue becomes 'group-leaderboard-tab')
2. then we toggle to Course Settings via Sidebar, and then disable the groupLeaderboardTitle
3. then we toggle back to Leaderboard via Sidebar

what it results, should the additional check not occur, will be the page for Group Leaderboard Tab.